tests/mgr/test_dashboard_url.py

```diff
@@ -0,0 +1,77 @@
+from cli.cephadm.cephadm import CephAdm
+from cli.utilities.utils import get_service_id
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class DashboardUrlError(Exception):
+    pass
+
+
+class CephMgrServiceFailError(Exception):
```

Comment:
instead use - https://github.com/red-hat-storage/cephci/blob/master/cli/exceptions.py#L32

Reply:
Done, added it.